CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * Fixed required URL checks in pubspec validation.
 * Fixed `pub outdated` execution - using specified package dir.
+* Various nit fixes to the generated report.
```

Comment:
Fix of deciding the latest version when. no version is given for a hosted package.